README.md

```diff
@@ -271,6 +271,36 @@ Add to your repository as `.github/workflows/sile.yaml`.
 This work flow assumes your project has a source file `my-document.sil` and will leave behind a `my-document.pdf`.
 Note the comments in [the section about Docker](#docker) regarding version tags.
 
+## Installing third-party packages
+
+Third-party SILE packages rely on the `luarocks` package manager.
+
+To install the latest version, for the given package, you may often use its provided “rockspec”:
+
+```bash
+# This example uses markdown.sile, a highly useful plugin that converts Markdown documents to SILE `.sil`.
+luarocks install markdown.sile --local
+```
+
+You may remove `--local`, in which case a global installation will be done (on most systems this requires `root`).
+
+If you use a per-system installation of your SILE modules, you will need to then run this before running SILE or will get errors involving missing modules:
+
+```bash
+eval $(luarocks path)
+```
+
+### Finding Lua version in use for running SILE
+
+To get your Lua version which is used for the execution of `sile`:
+
+```bash
+#!/bin/bash
+___LUA_VERSION=$(sile -e 'print(SILE.lua_version);os.exit()' 2> /dev/null)
```

Comment:
Maybe we should consider eventually having a `sile --lua-version`, to hide the weirdness of that one-liner Lua script.

Reply:
Maybe, but given the only use case is for where SILE is *not* using the system default I'm afraid there might be a lot of pitfalls to trying to simplify. What exactly are we trying to solve for?

(I cleaned up the usage and explanation a bit more, I'm willing to consider exposing something more precise if there is a demand and notable use case for it.)